recipes/python-intervaltree/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "2.1.0" %}
+
+package:
+  name: "python-intervaltree"
+  version: {{ version }}
+source:
+  url: https://files.pythonhosted.org/packages/ca/c1/450d109b70fa58ca9d77972b02f69222412f9175ccf99fdeaf167be9583c/intervaltree-2.1.0.tar.gz
+  sha256: aca5804b88f70cb49050c37b6de59090570f77a75aec1932966cf69f6a48810b
+build:
+  number: 0
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+  run:
+    - python
+test:
+  imports:
+    - intervaltree
+about:
+  home: https://pypi.org/project/intervaltree/#description
+  license: Apache
+  summary: "Faster than intervaltree_bio"
+extra:
+  web: https://github.com/chaimleib/intervaltree
```

Comment:
```
about:
    dev_uri: https://github.com/chaimleib/intervaltree
```

Reply:
Thanks. I'll add that. I have to add `python-sortedcontainers` too.

Reply:
Both packages have current versions in conda-forge.